_config.yml

```diff
@@ -1,7 +1,8 @@
 name: Azkaban
 safe: true
 lsi: false
-pygments: true
+# pygments: true
```

Comment:
Why not just remove it?
Having commented out code is not a good practice in general.